app/controllers/api/auth/stories_controller.rb

```diff
@@ -47,7 +47,7 @@ def create_resource
       story.creator_id = current_user.id
       story.account_id ||= story.series.try(:account_id)
       story.account_id ||= current_user.account_id
-      story.account_id ||= current_user.approved_accounts.first.try(:id)
+      story.account_id ||= current_user.approved_accounts.first.try(:id) # not sure if I should change this to look at token
```

Comment:
This is in the stories controller -- not sure whether to change this to just use the first account ID listed on the auth token, or perhaps the first account ID with role "admin", or else to leave it as is -- calling the User model, which joins against memberships. (Or should this simply be the user's `individual_account` ? )